lib/PSR/Storage/SQLite.hs

```diff
@@ -269,3 +279,35 @@ instance FromField EventType where
             "cancellation" -> pure Cancellation
             "selection" -> pure Selection
             _ -> returnError ConversionFailed f "Failed to parse event type"
+
+deriving newtype instance Aeson.ToJSON TraceLogs 
+deriving newtype instance Aeson.FromJSON TraceLogs 
+
+instance ToField TraceLogs where
+  toField traceLogs = toField $ Aeson.encode traceLogs
+
+instance FromField TraceLogs where
+  fromField f = do
+    fromField f >>= \s ->
+      case Aeson.decode $ BL.fromStrict $ T.encodeUtf8 s of
+        Just logs -> pure $ TraceLogs logs
+        Nothing -> returnError ConversionFailed f "Failed to parse trace logs"
+
+instance ToField ExUnits where
+  toField exUnits = toField $ Aeson.encode exUnits 
+
+instance FromField ExUnits where
+  fromField f = do
+    fromField f >>= \s ->
+      case Aeson.decode $ BL.fromStrict $ T.encodeUtf8 s of
+        Just exUnits -> pure exUnits 
+        Nothing -> returnError ConversionFailed f "Failed to parse execution units"
+
+-- sqlite-simple provides instances `FromRow` until 10 fields, so here we are
+instance (FromField a, FromField b, FromField c, FromField d, FromField e,
+          FromField f, FromField g, FromField h, FromField i, FromField j,
+          FromField k) =>
+    FromRow (a,b,c,d,e,f,g,h,i,j,k) where
+    fromRow = (,,,,,,,,,,) <$> field <*> field <*> field <*> field <*> field
+                          <*> field <*> field <*> field <*> field <*> field <*> field
+
```

Comment:
Can we use tuples of tuples instead? 😁 That might actually work for how we're actually using this instance. 

sqlite-simple defines the [`:.`](https://hackage.haskell.org/package/sqlite-simple-0.4.19.0/docs/Database-SQLite-Simple.html#t::.) type for combining together multiple FromRow instrances, which might help avoid needing this.